contracts/adapters/linkswap/LinkswapStakingAdapter.sol

```diff
@@ -0,0 +1,176 @@
+            totalRewards += StakingRewards(LSLP_YFL_WETH_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_DPI_LINK_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_MASQ_WETH_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_BUSD_LINK_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_GSWAP_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_CEL_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_YAX_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_CFI_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_USDC_POOL).earned(account,0);
+            totalRewards += StakingRewards(LSLP_LINK_USDT_POOL).earned(account,0);
+
+            return totalRewards;
+        } else if (token == CFI) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(CFI).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_LINK_CFI_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == MASQ) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(MASQ).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_MASQ_WETH_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == DPI) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(DPI).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_DPI_LINK_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == CEL) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(CEL).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_LINK_CEL_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == YAX) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(YAX).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_LINK_YAX_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == GSWAP) {
+            uint256 totalRewards = 0;
+            totalRewards += ERC20(GSWAP).balanceOf(account);
+            totalRewards += StakingRewards(LSLP_LINK_GSWAP_POOL).earned(account,1);
+            return totalRewards;
+        } else if (token == GOVERNANCE){
+            return ERC20(GOVERNANCE).balanceOf(account);
+        } else if (token == WETH) {
+            return ERC20(WETH).balanceOf(account);
+        } else if (token == LINK) {
+            return ERC20(LINK).balanceOf(account);
+        } else if (token == BUSD) {
+            return ERC20(BUSD).balanceOf(account);
+        } else if (token == USDC) {
+            return ERC20(USDC).balanceOf(account);
+        } else if (token == USDT) {
+            return ERC20(USDT).balanceOf(account);
```

Comment:
here, user's balances are returned

it shouldn't be here as only locked tokens should be returned

Reply:
Yes, this makes sense, thank you.